src/EnergyPlus/CurveManager.cc

```diff
@@ -3159,9 +3116,9 @@ namespace CurveManager {
             {
                 auto const SELECT_CASE_var(PerfCurve(CurveIndex).ObjectType);
```

Comment:
Since `ObjectType` is a string now, it would be better to use `CurveType` as the case variable. I know this only executes once, but don't like to add new string compares.

Reply:
I agree...and I struggled a bit with how this was working. This is one area where I couldn't figure out a clean solution without further refactoring. Really, we should make CurveInputN an array and loop through the number of independent variables to setup the appropriate output variables. I can take a crack at that.

Reply:
> Really, we should make CurveInputN an array and loop through the number of independent variables to setup the appropriate output variables.

Are you sure you want to do that here.  That could wait until the other part of the curve refactor.  For here, just simply using `CurveType` and the existing integers for that will be sufficient.

Reply:
There is no longer a `CurveType_TableMultiIV` variable (see [here](https://github.com/NREL/EnergyPlus/pull/6810/files/4d89755748773cfc357d0cbaf4f8e2cabc2c81fe#diff-6bab88f88cad01231c7611b162099cdaL148)). They generally weren't necessary and were just being used to do a [look up of a curve type string](https://github.com/NREL/EnergyPlus/pull/6810/files/4d89755748773cfc357d0cbaf4f8e2cabc2c81fe#diff-6bab88f88cad01231c7611b162099cdaL6065) for string comparisons. I was hoping to eliminate the use of the `CurveType` data member outside of the `PerformanceCurveObject` routine.

Reply:
Oh I see now - I hadn't caught on that curvetype and curveobjecttype weren't parallel.